src/Symfony/Component/Console/Application.php

```diff
@@ -1039,11 +1047,21 @@ private function findAlternatives($name, $collection)
     /**
      * Sets the default Command name.
      *
-     * @param string $commandName The Command name
+     * @param string $commandName   The Command name
+     * @param bool   $singleCommand Set to true if there is only one command is this Application
      */
-    public function setDefaultCommand($commandName)
+    public function setDefaultCommand($commandName, $singleCommand = false)
```

Comment:
In my opinion it should be called `$isSingleCommand` because `$singleCommand` looks like it's asking for the name of that single command.